packages/wallet/src/cip30.ts

```diff
@@ -71,7 +71,13 @@ export const createWalletApi = (
     logger.debug('getting balance');
     try {
       const wallet = await walletReady;
-      const value = await firstValueFrom(wallet.balance.available$);
+      // Review: not including balance.rewardAccounts.{deposit,rewards}, should we?
+      // Not sure what the intended use case of getBalance is.
+      // We should include it if:
+      // - If it's to display the balance to the user
+      // - If it's to get quantity of coins delegated (although not sure if deposit is included)
```

Comment:
Deposits are held in a separate ada pot, and are not counted as part of the user's delegated funds.